Projects/views.py

```diff
@@ -231,6 +231,8 @@ def response_pr(request):
             if pr.status==2:
                 pr.status=3
                 pr.from_user.profile.points=pr.from_user.profile.points+pr.issue.points+bonus_pts-deduct_pts
+                pr.from_user.profile.bonus_points += bonus_pts
+                pr.from_user.profile.deducted_points += deduct_pts
```

Comment:
the else part should also be handled on rejecting issues. profile.points will reduce but noty always to - points - issue.points
think about this :smile: